src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudConfig.scala

```diff
@@ -212,6 +213,7 @@ object FireCloudConfig {
   object ImportService {
     lazy val server: String = config.getString("importService.server")
     lazy val bucket: String = config.getString("importService.bucketName")
+    lazy val enabled: Boolean = config.optionalBoolean("importService.enabled").getOrElse(true)
```

Comment:
not used in this PR. I'm envisioning at a future point we'll want to turn off Import Service in code but leave it running for easy rollback.